File: openapi-test-kotlin-server-quarkus-reactive-client-mp-reactive/testapi/model/OptionalInfo.kt
package testapi.model

import com.fasterxml.jackson.annotation.JsonProperty
import java.util.Optional

data class OptionalInfo(
  @JsonProperty("optional_field") val optional_field: Optional<String>,
  @JsonProperty("optional_with_default") val optional_with_default: Optional<String>,
  @JsonProperty("required_field") val required_field: Optional<String>
)

File: openapi-test-scala-server-spring-client-jdk/testapi/model/OptionalInfo.scala
package testapi.model

import com.fasterxml.jackson.annotation.JsonProperty
import java.util.Optional

case class OptionalInfo(
  @JsonProperty("optional_field") optional_field: Optional[String],
  @JsonProperty("optional_with_default") optional_with_default: Optional[String],
  @JsonProperty("required_field") required_field: Optional[String]
)